analyzers/rspec/cs/S5753_c#.html

```diff
@@ -40,8 +50,21 @@ <h2>Compliant Solution</h2>
   ...
 }
 </pre>
+<p>At application level, configured in the Web.config file:</p>
+<pre>
+&lt;configuration&gt;
+   &lt;system.web&gt;
+      &lt;pages validateRequest="true" /&gt;
+      ...
+      &lt;!-- see https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.httpruntimesection.requestvalidationmode?view=netframework-4.8 --&gt;
```

Comment:
This link is here with `see` as well as on the bottom in `see` section. I think we can remove one of them (this one)